### PR TITLE
Add calculations logic - Milestone 4

### PR DESCRIPTION
**Pull request by MauricioRobayo**

### CODE REVIEW

File: src/components/App.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+import Display from './Display'
+import ButtonPanel from './ButtonPanel'
+import calculate from './../logic/calculate' // eslint-disable-line
```
Comment (shloch) [inline src/components/App.js:4]: i guess it will work too with
`import calculate from '../logic/calculate'`

File: src/logic/operate.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import Big from 'big.js/big'
+
+const operations = {
+  '+': (a, b) => a.plus(b),
+  '-': (a, b) => a.minus(b),
+  '*': (a, b) => a.times(b),
+  '/': (a, b) => a.div(b),
```
Comment (shloch) [inline src/logic/operate.js:7]: dont forget to manage division by ZERO when using this component

**Review by shloch — APPROVED** [review]
## Nice job @MauricioRobayo 😊

minor observations before

### This milestone is considered valid, you can merge this PR 😊😊

File: src/components/App.js (hunk 1 above)
Comment (MauricioRobayo) [reply src/components/App.js:4]: That's right, I'll fix that 😄 .

File: src/logic/operate.js (hunk 2 above)
Comment (MauricioRobayo) [reply src/logic/operate.js:7]: Thanks! I'll take that into account.

**Pull request merged**